core/src/test/scala/org/apache/spark/deploy/history/FsHistoryProviderSuite.scala

```diff
@@ -1256,12 +1257,7 @@ class FsHistoryProviderSuite extends SparkFunSuite with Matchers with Logging {
     val cstream = codec.map(_.compressedOutputStream(fstream)).getOrElse(fstream)
     val bstream = new BufferedOutputStream(cstream)
     if (isNewFormat) {
```

Comment:
We don't support the old log format anymore, so you could actually remove this parameter since you're here...